js/reviewApi.js

```diff
@@ -1,27 +1,143 @@
-	//Review Api call
-
-	$(document).ready(function(){
-		var settings = {
-			"url": "/reviews",
-			"method": "GET",
-			"timeout": 0,
-		  };
-
-		  $.ajax(settings)
-		  .done(function (response) {
-			  console.log(response);
-			  response.map(function(r){
+function getAllReviews() {
+	var settings = {
+		"url": "/reviews",
+		"method": "GET",
+		"timeout": 0,
+	};
+
+	$.ajax(settings)
+		.done(function (response) {
+			console.log(response);
+			response.map(function (r) {
 				$("#apiReview").append(`
-					<div data-review_id=${r.id}>
-					    <p>${r.user.email} : </p>
-						<p>${r.score}</p>
-						<p>${r.comment}</p>
-					</div>
-				`)
-			  })
+			<div data-review_id=${r.id}>
+			<p>${r.user.email} : </p>
+			<p>${r.score}</p>
+			<p>${r.comment}</p>
+			</div>
+			`)
+			})
+			console.log(response);
+		})
+		.fail(function (err) {
+			console.error(err)
+		});
+}
+
+function getUserByEmail(userEmail) {
+	// setting for oue getUserByEmail route
+	var getUserSettings = {
+		"url": "/user/" + userEmail,
+		"method": "GET"
+	}
+	var userID;
+	// check user route by email to see if the user exists
+	$.ajax(getUserSettings)
+		.done(function (response) { // successful response
+			console.log("success", response);
```

Comment:
I can see in the console.log that the call is able to confirm and see that the user exists when the review form is given an email already in the db, however it then prompts the user to the create user form again.

Reply:
@margeschrec do you have some time this evening we can go over this and try to debug it?

Reply:
yes, I have time